docs/manual/developer_guide.adoc

```diff
@@ -1176,7 +1176,12 @@ fi
 
 When writing new bash remediations content, please follow the following guidelins:
 
-* Use tabs for indentation rather than spaces.
+* Use four spaces for indentation rather than tabs.
+* You can use macros from `shared/macros-bash.jinja` in the remediation content.
+If the macro is used from a nested block, use the `indent` jinja2 filter assuming the 4-space indentation.
+Typically, you want to call the macro with the intended indentation, and as `indent` doesn't indent the first line by default, you just pass the number of spaces as the only argument.
+
+See the remediation for rule `ensure_fedora_gpgkey_installed` for reference.
```

Comment:
This has destroyed the rest of the bullet list. Please fix this to make the list displayed as a list.